daemon/errors.go

```diff
@@ -21,4 +21,6 @@ var (
 	ErrUnauthorized = errors.NewKind("unauthorized: authentication required to access %s (image: %s)")
 	// ErrLanguageDetection indicates that language was not detected by Enry.
 	ErrLanguageDetection = errors.NewKind("could not autodetect language")
+	// ErrInvalidEncoding is returned for parse requests that provide a file in non-UTF8 encoding.
```

Comment:
I'm not sure we want to say that other encodings are "invalid", since in this case the file is not broken, it's merely not using the encoding we want. I worry the user might think we're saying the problem is in their data rather than a limitation of our system.

I suggest being more specific about the complaint here, e.g., `ErrUnknownEncoding = errors.NewKind("unknown source file encoding (expected UTF-8)")` or words to that effect. (After all, if we knew what it was, we could have transcoded it)